test_scripts/Polices/External_UCS/External_Consent_OFF/028_ATF_Policies_External_Consent_OFF_same_notification_user_consent_allowed_externalConsentStatus_disallowed.lua

```diff
@@ -0,0 +1,134 @@
+--------------------------------------Requirement summary---------------------------------------------
+-- [Policies] External UCS: externalConsentStatus vs. consentedFunctions priority
+
+------------------------------------General Settings for Configuration--------------------------------
+require('user_modules/all_common_modules')
+local common_functions_external_consent = require('user_modules/ATF_Policies_External_Consent_common_functions')
+local common_steps = require('user_modules/common_steps')
+local common_functions = require ('user_modules/common_functions')
+
+---------------------------------------Common Variables-----------------------------------------------
+local id_group_1
+local policy_file = config.pathToSDL .. "storage/policy.sqlite"
+
+---------------------------------------Preconditions--------------------------------------------------
+-- Start SDL and register application
+common_functions_external_consent:PreconditonSteps("mobileConnection","mobileSession")
+-- Activate application
+common_steps:ActivateApplication("Activate_Application_1", config.application1.registerAppInterfaceParams.appName)
```

Comment:
@maria-mih , We can use simple code common_steps:PreconditionSteps("Preconditions", 7) to replace for 
```
+-- Start SDL and register application
+common_functions_external_consent:PreconditonSteps("mobileConnection","mobileSession")
+-- Activate application
+common_steps:ActivateApplication("Activate_Application_1", config.application1.registerAppInterfaceParams.appName)
```